src/Illuminate/Collections/Collection.php

```diff
@@ -785,6 +785,32 @@ public function only($keys)
         return new static(Arr::only($this->items, $keys));
     }
 
+    /**
+     * Get the items that are not filled.
+     *
+     * @param  mixed  $keys
+     * @return static
+     */
+    public function blank()
+    {
+        return $this->filter(function ($item) {
+            return blank($item);
+        });
```

Comment:
optionally this may be shortened with:
```suggestion
        return $this->filter('blank');
```
same for `filled()` below